app/src/main/java/org/glucosio/android/analytics/GoogleAnalytics.java

```diff
@@ -28,15 +34,19 @@ public void init(@NonNull final Context context) {
 
     @Override
     public void reportScreen(@NonNull String name) {
-        mTracker.setScreenName(name);
-        mTracker.send(new HitBuilders.ScreenViewBuilder().build());
+        if (enabled) {
```

Comment:
Even if reportScreen doesn't work, Analytics could send Google other statistics about the app.
We may consider using `setAppOptOut(true)` to disable Analytics at app-level.